### PR TITLE
Updates to sparse feature vectors

### PR DESCRIPTION
**Pull request by Sidu28**
Updated the sparse vectors to reflect the new features that kevin added.  features_list.csv contains all the possible features which is static for now.

### CODE REVIEW

File: feature_utils.py [hunk 1]
```diff
@@ -35,6 +35,12 @@
   }
 """
 
+def get_features():
```
Comment (mananshah99) [inline feature_utils.py:38]: Perhaps rename to get_feature_names?

**Review by mananshah99 — APPROVED** [review]
LGTM, one minor comment

**Comment by mananshah99** [comment]
Quick question: does this change allow me to access sparse feature vectors of fixed dimension for the database?

**Comment by Sidu28** [comment]
@mananshah99 Yes it does.  The feature_utils.extract_features function will return two values now, the second of which is your fixed-length sparse vector.  It will be around 7,000 in length

**Review by kevinwho777 — APPROVED** [review]
Looks good!

**Pull request merged**